sa/storage-authority.go

```diff
@@ -897,6 +887,25 @@ func addFQDNSet(tx *gorp.Transaction, names []string, serial string, issued time
 	})
 }
 
+type execable interface {
+	Exec(string, ...interface{}) (sql.Result, error)
+}
+
+func addIssuedNames(tx execable, cert *x509.Certificate) error {
+	var qmarks []string
+	var values []interface{}
+	for _, name := range cert.DNSNames {
+		values = append(values,
+			core.ReverseName(name),
+			core.SerialToString(cert.SerialNumber),
+			cert.NotBefore)
+		qmarks = append(qmarks, "(?, ?, ?)")
```

Comment:
Why not `strings.Repeat("(?, ?, ?), ", len(values)` then slice off the last 2 characters?